fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -575,7 +650,7 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 		}
 	}
 
-	Future<Void> getError() override { return errorPromise.getFuture(); }
+	Future<Void> getError() override { return std::move(errorFuture); }
```

Comment:
AFAICT, `Future` doesn't have a move constructor, so this is just a more verbose way of copying. Also, I would be concerned with making this method unsafe to call more than once if a move constructor were ever implemented for `Future`.

Reply:
Makes sense. Fixed applied.